src/__tests__/create-ssr-search-client.spec.ts

```diff
@@ -35,4 +35,49 @@ describe('createSSRSearchClient', () => {
       `angular-instantsearch-server (${VERSION})`
     );
   });
+
+  it('forwards the default options to the search client', () => {
+    const addAlgoliaAgent = jest.fn();
+    algoliasearch.mockImplementation(() => {
+      return {
+        addAlgoliaAgent,
+      };
+    });
+
+    const ssrSearchClient = createSSRSearchClient({
+      appId: 'appId',
+      apiKey: 'apiKey',
+      httpClient: null,
+      HttpHeaders: null,
+      makeStateKey: null,
+      transferState: null,
+    });
+
+    expect(addAlgoliaAgent).toHaveBeenCalledWith('appId', 'apiKey', {});
```

Comment:
We expect the search client (called `algoliasearch`) to be called here:

```suggestion
    expect(algoliasearch).toHaveBeenCalledWith('appId', 'apiKey', {});
```